findspam.py

```diff
@@ -1142,7 +1142,10 @@ class FindSpam:
         {'method': mostly_dots, 'all': True, 'sites': ['codegolf.stackexchange.com'],
          'reason': 'mostly dots in {}', 'title': True, 'body': True, 'username': False, 'body_summary': False,
          'stripcodeblocks': False, 'max_rep': 50, 'max_score': 0},
-
+        # Title ends with Comma (IPS Troll)
+        {'regex': r".*\,$", 'all': False, 'sites': ["interpersonal.stackexchange.com"],
+         'reason': "title ends with comma", 'title': True, 'body': False, 'username': False, 'stripcodeblocks': False,
+         'body_summary': True, 'max_rep': 50, 'max_score': 0},
```

Comment:
`body_summary` is the first 300 or so characters of the body, which we get from the realtime socket. I don't think we want to scan that for this rule, so it can be `False`.

Reply:
Done. Also made it for all sites on the network since you appropriately said it's accurate enough for that.